indico/modules/rb/client/js/common/rooms/RoomDetailsModal.jsx

```diff
@@ -106,6 +113,14 @@ export default connect(
     attributes: roomsSelectors.getAttributes(state, {roomId}),
   }),
   dispatch => ({
+    gotoAllBookings(roomName) {
+      const params = {
+        text: roomName,
+      };
+      dispatch(globalActions.resetPageState('calendar'));
+      dispatch(filtersActions.setFilters('calendar', {text: roomName}, false));
+      dispatch(pushRoute(`/calendar?${qs.stringify(params)}`));
```

Comment:
What about setting the month view mode? Redirecting to the day view displays less information than in the room details.

Reply:
Yeah, I'll do it if there's a way. Redirecting to calendar page with mode=months in the url doesn't work